docs/reference/remap/get_env_var.cue

```diff
@@ -0,0 +1,31 @@
+package metadata
+
+remap: functions: get_env_var: {
+	arguments: [
+		{
+			name:        "name"
```

Comment:
```suggestion
			name:        "value"
```

Right now, we have the convention of naming the first argument `value`. It's something we could change in the future, but for now, let's stay consistent.

Reply:
I think in this case, calling the environment variable name `value` is a little confusing. My preference is for `name` here still.

Reply:
Hmm, yeah, that's a good point. 👍